### PR TITLE
Address Colin nits

### PR DESCRIPTION
**Pull request by chris-wood**

### CODE REVIEW

**Review by dconnolly — APPROVED** [review]

File: draft-irtf-cfrg-ristretto255-decaf448.md [hunk 1]
```diff
@@ -322,7 +323,7 @@ Since ristretto255 is a prime-order group, every element except the
 identity is a generator, but for interoperability a canonical generator
 is selected, which can be internally represented by the Curve25519
 basepoint, enabling reuse of existing precomputation for scalar
-multiplication. This is its encoding (#encoding255):
+multiplication. This is its encoding using the method described in (#encoding255):
```
Comment (str4d) [inline draft-irtf-cfrg-ristretto255-decaf448.md]:
```suggestion
multiplication. This is its encoding as produced by the function
specified in (#encoding255):
```

File: draft-irtf-cfrg-ristretto255-decaf448.md [hunk 2]
```diff
@@ -585,7 +586,7 @@ Since decaf448 is a prime-order group, every element except the
 identity is a generator, but for interoperability a canonical generator
 is selected, which can be internally represented by the edwards448
 basepoint, enabling reuse of existing precomputation for scalar
-multiplication. This is its encoding (#encoding448):
+multiplication. This is its encoding using the method described in (#encoding448):
```
Comment (str4d) [inline draft-irtf-cfrg-ristretto255-decaf448.md]:
```suggestion
multiplication. This is its encoding as produced by the function
specified in (#encoding448):
```

File: draft-irtf-cfrg-ristretto255-decaf448.md [hunk 3]
```diff
@@ -486,8 +487,8 @@ internal representation and to the internal representation obtained by
 encoding and then re-decoding it. However, the internal
 representations themselves might not be identical.
 
-Implementations **MAY** also perform byte comparisons on encodings for
-an equivalent, although less efficient, result.
+Implementations **MAY** also perform byte comparisons on encoded
+representations elements for an equivalent, although less efficient, result.
```
Comment (str4d) [inline draft-irtf-cfrg-ristretto255-decaf448.md]:
```suggestion
Implementations **MAY** also perform byte comparisons on the encodings
of group elements (produced by (#encoding255)) for an equivalent, although
less efficient, result.
```

File: draft-irtf-cfrg-ristretto255-decaf448.md [hunk 4]
```diff
@@ -721,8 +722,8 @@ internal representation and to the internal representation obtained by
...
```
Comment (str4d) [inline draft-irtf-cfrg-ristretto255-decaf448.md]:
```suggestion
Implementations **MAY** also perform byte comparisons on the encodings
of group elements (produced by (#encoding448)) for an equivalent, although
less efficient, result.
```

**Review by str4d — CHANGES_REQUESTED** [review]

**Comment by chris-wood** [comment]
Thanks @str4d 😃 I applied all your suggestions.

**Review by str4d — APPROVED** [review]

**Pull request merged**